extensions/amp-social-share/1.0/social-share.js

```diff
@@ -160,10 +168,18 @@ function handleActivation(finalEndpoint, target) {
         `Could not complete system share.  Navigator unavailable. ${NAME}`
       );
     }
-  } else if (protocol === 'sms') {
+  } else if (protocol === 'sms' || protocol === 'mailto') {
```

Comment:
This looks like a behavior change for `mailto` -- what was the motivation here?

Reply:
In the version prior to this PR, this was handled in the Amp component and the correct `target` is passed into the Preact component.  However, we also want to do this check within the Preact component as well (in case the user is not using Amp).

So essentially we want to do duplicate the behavior from AMP over into the Preact component.  Here's the link to the same logic on the amp side.

Reply:
https://github.com/ampproject/amphtml/blob/master/extensions/amp-social-share/1.0/amp-social-share.js#L141